hops/featurestore_impl/featureframes/FeatureFrame.py

```diff
@@ -211,8 +212,8 @@ def __init__(self, **kwargs):
         Initialize featureframe state using the parent class constructor
         """
         super(TFRecordsFeatureFrame, self).__init__(**kwargs)
-
-
+        self._spark_tfrecord_format = kwargs["data_format"]
```

Comment:
I think after the `super().__init__()` is called, the class already has the attribute `data_format` since the init of the base class just does a `self.__dict__.update(kwargs)`. In case you want to simplify it